### PR TITLE
Fixed the storage cmdlets alias not in resource mode bug

### PR DESCRIPTION
**Pull request by blueww**
Fixed the storage cmdlets alias not in resource mode bug

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@blueww__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (weiwei). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci test this please

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by hovsepm** [comment]
on-demand job: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/378/

**Comment by hovsepm** [comment]
sign job: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-sign/181/

File: setup/azurecmdfiles.wxi [hunk 1]
```diff
@@ -8,6 +8,9 @@
                         <Component Id="cmp0761C025FCF8EF027A5BC23C72777C52" Guid="*">
                             <File Id="fil80798000B169AC6D62DD8331A55CF6FF" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\Azure.Storage\Azure.Storage.psd1" />
                         </Component>
+                        <Component Id="cmpCC0D4240D3640D2D425852E0F2A4D31E" Guid="*">
```
Comment (markcowl) [inline setup/azurecmdfiles.wxi]: This change should not be added - the shortcut should not be added to ASM cmdlets

File: setup/azurecmdfiles.wxi [hunk 2]
```diff
@@ -4413,6 +4419,9 @@
                         <Component Id="cmp74B0B7D2461EE67BC1B2C706CD41372F" Guid="*">
                             <File Id="filD84EC14AB325E2924B8FFE3A4D3D018A" KeyPath="yes" Source="$(var.sourceDir)\ServiceManagement\Azure\Sql\Azure.Storage.psd1" />
                         </Component>
+                        <Component Id="cmp572CBAA060A870457B636480C182F6E3" Guid="*">
```
Comment (markcowl) [inline setup/azurecmdfiles.wxi]: Same comment

**Comment by markcowl** [comment]
@EmmaZhu Can you review thgis contribution?

File: setup/azurecmdfiles.wxi (hunk 1 above)
Comment (EmmaZhu) [inline setup/azurecmdfiles.wxi]: It's a script to define cmdlet aliases.Is there any particular reason that we cannot call script when loading a module?

Comment (markcowl) [inline setup/azurecmdfiles.wxi]: No, I'm saying the script should only be copied into the folder you need it, not into 3 folders

**Pull request merged**